playbooks/hosts/amc_production/group_vars/edx

```diff
@@ -0,0 +1,204 @@
+---
+
+EDXAPP_MONGO_HOSTS: ['prod-amc-mongo-1', 'prod-amc-mongo-2', 'prod-amc-mongo-3']
+EDXAPP_MONGO_USER: 'edxapp'
+EDXAPP_MONGO_PASSWORD: "{{ secret_EDXAPP_MONGO_PASSWORD }}"
+EDXAPP_MONGO_DB_NAME: 'edxapp'
+EDXAPP_MYSQL_HOST: '{{ _mysql_host }}'
+
+
+EDXAPP_LMS_ENV: 'lms.envs.amc'
+EDXAPP_CMS_ENV: 'cms.envs.amc'
+
+EDXAPP_AUTH_EXTRA:
+  APPSEMBLER_SECRET_KEY: "{{secret_APPSEMBLER_SECRET_KEY}}"
+  APPSEMBLER_AMC_API_BASE: "https://amc.appsembler.com/api"
+  MAILGUN_ACCESS_KEY: "{{secret_MAILGUN_ACCESS_KEY}}"
+  MAILGUN_SERVER_NAME: "{{secret_MAILGUN_SERVER_NAME}}"
+
+edx_platform_repo: "https://github.com/appsembler/edx-platform.git"
+edx_ansible_source_repo: "https://github.com/appsembler/configuration"
+
+EDX_VERSION: named-release/dogwood
+edx_platform_version: "appsembler/amc/develop"
+configuration_version: "{{ EDX_VERSION | default('named-release/dogwood.rc') }}"
+certs_version: "{{ EDX_VERSION | default('named-release/dogwood.rc') }}"
+forum_version: "{{ EDX_VERSION | default('named-release/dogwood.rc') }}"
+xqueue_version: "{{ EDX_VERSION | default('named-release/dogwood.rc') }}"
+INSIGHTS_VERSION: "{{ EDX_VERSION | default('named-release/dogwood.rc') }}"
+
+EDXAPP_FEATURES:
+  AUTH_USE_OPENID_PROVIDER: true  # default: true
+  ENABLE_OAUTH2_PROVIDER: true  # default: false
+  ALLOW_ALL_ADVANCED_COMPONENTS: true  # default: false
+  ENABLE_SYSADMIN_DASHBOARD: True  # default: false
+  #ENABLE_CREATOR_GROUP: true  # default: false
+  CERTIFICATES_ENABLED: true  # default: true
+  ENABLE_DISCUSSION_SERVICE: true  # default: true
+  ENABLE_INSTRUCTOR_ANALYTICS: false  # default: false
+  SUBDOMAIN_BRANDING: false  # default: false
+  SUBDOMAIN_COURSE_LISTINGS: false  # default: false
+  PREVIEW_LMS_BASE: "{{ EDXAPP_PREVIEW_LMS_BASE }}"
+  ENABLE_S3_GRADE_DOWNLOADS: true  # default: true
+  USE_CUSTOM_THEME: "{{ edxapp_use_custom_theme }}"
+  ENABLE_MKTG_SITE: "{{ EDXAPP_ENABLE_MKTG_SITE }}"  # default: false
+  AUTOMATIC_AUTH_FOR_TESTING: "{{ EDXAPP_ENABLE_AUTO_AUTH }}"  # default: false
+  ENABLE_THIRD_PARTY_AUTH: "{{ EDXAPP_ENABLE_THIRD_PARTY_AUTH }}" # default: false
+  ENABLE_VIDEO_UPLOAD_PIPELINE: false  # default: false
+  ENABLE_DISCUSSION_HOME_PANEL: true  # default: true
+  ENABLE_CONTENT_LIBRARIES: true  # default: true
+  #search features in CMS
+  ENABLE_COURSEWARE_INDEX: true  # default: false
+  ENABLE_LIBRARY_INDEX: true  # default: false
+  #search features in LMS
+  ENABLE_COURSEWARE_SEARCH: true  # default: false
+  ENABLE_DASHBOARD_SEARCH: true  # default: false
+  ENABLE_COURSE_DISCOVERY: false # default: false
```

Comment:
I'm not sure anymore, but I think because each test drive user only gets one course so it doesn't make sense to have course discovery enabled. Also, I think I remember it causing us problems with user being able to see other users courses.